ddtrace/profiling/exporter/pprof.py

```diff
@@ -241,12 +241,18 @@ def _get_trace_id(event):
             return str(list(sorted(event.trace_ids))[0])
         return ""
 
+    @staticmethod
+    def _get_thread_name(thread_id, thread_name):
+        if thread_name is None:
+            return "Anonymous Thread %d" % thread_id
```

Comment:
Would "Unnamed" be a better choice than "Anonymous"? "Anonymous" might be interpreted as being intentionally unnamed. Not a big deal though 🤷‍♂️

Reply:
This is unrelated to the PR (it has been like that for a year) ;)